<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LANStartClientDialog</class>
 <widget class="QDialog" name="LANStartClientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>LANゲームに参加 - melonDS</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>プレイヤー名:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtPlayerName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="tvAvailableGames">
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbDirectConnect">
     <property name="title">
      <string>直接接続</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_direct">
      <item>
       <widget class="QLabel" name="label_host">
        <property name="text">
         <string>ホストアドレス:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtDirectHost">
        <property name="placeholderText">
         <string>例: 192.168.1.100</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_directport">
        <property name="text">
         <string>ポート:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbDirectPort">
        <property name="minimum">
         <number>1024</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>7064</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDirectConnect">
        <property name="text">
         <string>接続</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;LAN検出はローカルネットワークのみです。&lt;/p&gt;&lt;p&gt;インターネット対戦は「直接接続」を使用してください。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LANStartClientDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LANStartClientDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
